bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/gmos/GmosOiwfsProbeArm.scala

```diff
@@ -0,0 +1,142 @@
+package edu.gemini.spModel.gemini.gmos
+
+import java.awt.Shape
+import java.awt.geom.{Point2D, AffineTransform}
+
+import edu.gemini.pot.ModelConverters._
+import edu.gemini.shared.util.immutable.ImPolygon
+import edu.gemini.skycalc.Offset
+import edu.gemini.spModel.core.Coordinates
+import edu.gemini.spModel.inst.{ArmAdjustment, ProbeArmGeometry}
+import edu.gemini.spModel.obs.context.ObsContext
+import edu.gemini.spModel.telescope.IssPort
+
+object GmosOiwfsProbeArm extends ProbeArmGeometry {
+  import edu.gemini.spModel.inst.FeatureGeometry.transformPoint
+
+  val instance = this
```

Comment:
In the worst case you can write a Java interface that provides the API you want to see from Java, then implement it with a Scala class that your Java code can instantiate and use. Depends on your situation; may not be worth it.